R/04-webprocess-obj.R

```diff
@@ -83,6 +97,7 @@ setMethod(f="initialize",signature="webprocess",
             .Object@url <- url
             .Object@email <- email
             .Object@wait <- wait
+            .Object@sleep.time <- sleep.time
             
             # // -- supporting pass through of existing inputs arguments *when* they are applicable.
             old.inputs = inputs(.Object)
```

Comment:
As an alternative to setting the defaults from `gconfig` in `initialize()`, you could set them below in `webprocess()` (at L138, https://github.com/USGS-R/geoknife/pull/239/files#diff-efe130966420f1f104699b8d36f6476aR138) like this:
```r
 setMethod("webprocess", signature('missing'), function(.Object, url=gconfig('wps.url'), 
    sleep.time=gconfig('sleep.time'), wait=gconfig('wait'), email=gconfig("email"), ...) {
   ## create new webprocess object
   webprocess <- new("webprocess", url=url, sleep.time=sleep.time, wait=wait, email=email, ...)
   return(webprocess)
 })
```
I think that just requires that your users won't be doing `new('webprocess' on their own)` - and I think they shouldn't, any more than they'd dare to use a webprocess without calling `initialize()`